src/components/ProfileValidation/TermsAndConditionsWrapped.tsx

```diff
@@ -0,0 +1,41 @@
+import { Box } from "@mui/material"
```

Comment:
I'm wrapping terms and conditions control to hack the style to be more acceptable